vmdb/app/models/ems_microsoft/powershell.rb

```diff
@@ -38,6 +38,10 @@ def log_dos_error_results(results)
         log_header = "MIQ(#{self.class.name}##{__method__})"
         error = results[:data].collect { |d| d[:stderr] }.join
         $scvmm_log.error("#{log_header} #{error}") unless error.blank?
+
+        if error.downcase.include? "'get-scvmmserver' is not recognized"
+          raise "Hyper-V Server is not directly supported. "
+        end
```

Comment:
Thinking maybe the post-processing of results[:data] should move into it's own method that just extracts the error.